src/Symfony/Component/AssetMapper/ImportMap/ImportMapConfigReader.php

```diff
@@ -116,4 +129,20 @@ public function getRootDirectory(): string
     {
         return \dirname($this->importMapConfigPath);
     }
+
+    private function extractVersionFromLegacyUrl(string $url): ?string
+    {
+        // URL pattern https://ga.jspm.io/npm:bootstrap@5.3.2/dist/js/bootstrap.esm.js
+        $lastAt = strrpos($url, '@');
+        if (false === $lastAt) {
```

Comment:
I'll ever stop suggesting those CS changes ;P
```suggestion
        if (false === $lastAt = strrpos($url, '@')) {
```